### PR TITLE
[Template] Refactor issue template

### PR DESCRIPTION
**Pull request by Harold-lkk**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/4-documentation.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+description: Report an issue related to https://mmocr.readthedocs.io/en/latest/.
+labels: "docs"
+title: "[Docs] "
+
+body:
+- type: textarea
+  attributes:
+    label: 📚 The doc issue
+    description: >
+      A clear and concise description of what content in https://mmocr.readthedocs.io/en/latest/ is an issue.
```
Comment (gaotongxiao) [inline .github/ISSUE_TEMPLATE/4-documentation.yml:11]:
```suggestion
      A clear and concise description of what content in https://mmocr.readthedocs.io/en/latest/ or https://mmocr.readthedocs.io/en/dev-1.x/ is an issue.
```

**Comment by gaotongxiao** [comment]
@zengyh1900

File: .github/ISSUE_TEMPLATE/1-bug-report.yml [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+      label: Prerequisite
+      description: Please check the following items before creating a new issue.
+      options:
+      - label: I have searched [Issues](https://github.com/open-mmlab/mmocr/issues) and [Discussions](https://github.com/open-mmlab/mmocr/discussions) but cannot get the expected help.
+        required: true
+      # - label: I have read the [FAQ documentation](https://mmocr.readthedocs.io/en/1.x/notes/4_faq.html) but cannot get the expected help.
+      #   required: true
+      - label: The bug has not been fixed in the [latest version](https://github.com/open-mmlab/mmocr).
+        required: true
+
```
[the remark this answers is not in the document]
Comment (zengyh1900) [reply .github/ISSUE_TEMPLATE/1-bug-report.yml:27]: An example: https://github.com/open-mmlab/mmediting/pull/1286/files#diff-7ef8d00ae79e1c2ac42d02e7069549d69e2324e6cf37f8d4373428f360091babR39-R50

File: .github/ISSUE_TEMPLATE/4-documentation.yml (hunk 1 above)
Comment (zengyh1900) [reply .github/ISSUE_TEMPLATE/4-documentation.yml:11]: We can also use a single option item to label main/dev-1.x 
An example: https://github.com/open-mmlab/mmediting/pull/1286/files#diff-ffcbbc5f70e05184cd63fe22a18df4180c461ca4cf2dc99bb3251e22a6505aa1R7-R17

File: .github/ISSUE_TEMPLATE/1-bug-report.yml [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+    validations:
+      required: true
+
+  - type: dropdown
+    id: branch
+    attributes:
+      label: Branch
+      description: The problem arises when I'm working on
+      options:
+        - master branch https://github.com/open-mmlab/mmocr
```
Comment (gaotongxiao) [inline .github/ISSUE_TEMPLATE/1-bug-report.yml:44]:
```suggestion
        - main branch https://github.com/open-mmlab/mmocr
```

File: .github/ISSUE_TEMPLATE/2-feature_request.yml [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    attributes:
+      label: Any other context?
+      description: |
+        Have you considered any alternative solutions or features? If so, what are they? Also, feel free to add any other context or screenshots about the feature request here.
+
+  - type: markdown
+    attributes:
+      value: |
+        ## Acknowledgement
+        Thanks for taking the time to fill out this report.
```
Comment (gaotongxiao) [inline .github/ISSUE_TEMPLATE/2-feature_request.yml:34]:
```suggestion
        Thanks for taking time to fill out this report.
```

**Review by gaotongxiao — APPROVED** [review]

**Pull request merged**